source/_includes/installation/operating_system.md

```diff
@@ -91,6 +91,21 @@ To use this method, follow the steps described in the procedure below: [Write th
 
 - To use this method, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, follow the steps described in the procedure below: [Write the image to your boot media](#write-the-image-to-your-boot-media).
 
+<details>
```

Comment:
Aa right, that is also a details thingy 😄 

I meant `{% details "title" %} ...content... {% enddetails %}`

Reply:
Awesome, did not know about this, it looks much better now